### ISSUE TITLE
Quick select hang up reason.

### ISSUE THREAD

**Issue by lampholder** [element-hq/element-call#271]
Sometimes I hang up at the end of a call. No reason needed there.

Sometimes I have to hang up mid call. When I do, it would almost always benefit from my being able to send a message to folks saying why I had to quit:

1. My connection was so bad I couldn't meaningfully participate in this meeting any more (by implication I may be rejoining if I can sort my connectivity issues)
2. I have another commitment starting now (by implication I will not be returning to this meeting)
3. I've had to drop temporarily - back in a sec

The option to send a reason should come after you've left the room - we shouldn't introduce an additional click to leave (I'm looking at you, zoom).

As an alternative I could just try and interrupt the person speaking to say why I'm leaving, but this is disruptive, especially if I'm dropping due to connectivity issues.

**Comment by daniellekirkwood** [comment]
I don't think we would plan to implement this -- ideally we have chat and users can type "brb" or "doorbell" just as quickly as selecting some pre-defined things when leaving a call... 

if this is still a need, let's talk about it!
